### ISSUE TITLE
Form.Width returns incorrect value

### ISSUE THREAD

**Issue by dmrvb** [twinbasic/twinbasic#1837]
**Describe the bug**
Form.width and Form.scalewidth return the same value, (and similar issue for heights).
I would expect different values due to borders and/or titlebar (as in VB6).

it looks like .scalewidth is actually returning .width

**To Reproduce**
Create a Form 
Add a button

 Private Sub Command1_Click()
     Debug.Print "w " ; Width, ScaleWidth
     Debug.Print "h " ; Height, ScaleHeight
 End Sub

win11 tB515

I also note that the height/width values are slightly different to those when I compile a similar form in VB6 and run the EXE in win11 with a commoncontrols6 manifest. I will investigate this more when this bug is resolved, but the differences are small so its not a big issue.

**Comment by WaynePhillipsEA** [comment]
Yes, I am aware of this one (although I'm not sure whether we actually had a bug report open for it).  Thanks
